src/simtools/applications/production_scale_events.py

```diff
@@ -119,66 +106,22 @@ def _parse(label, description):
         default=("prod6_LaPalma-{zenith}deg_gamma_cone.N.Am-4LSTs09MSTs_ID0_reduced.fits"),
```

Comment:
For all parameters: do not add the default value to the help message. argpars is printing it out (now it is duplicated).

Reply:
fixed